tabbing/index.js

```diff
@@ -0,0 +1,20 @@
+module.exports = {
+	init: function (flags) {
+
+		if (flags.get('a11yKeyboardFocus')) {
+			const container = document.querySelector('.js');
```

Comment:
As far as checking for JS folks goes, I was thinking of just handling that [in the CSS](https://github.com/Financial-Times/n-ui-foundations/blob/9e66637b170f83755c93bbdb8064ce7854ac30d4/util/main.scss#L67), where 
```
.has-tabbed {
```
would become
```
.no-js,
.has-tabbed {
```